addons/paint-snap/userstyle.css

```diff
@@ -50,6 +50,7 @@
   bottom: calc(100% + 22px);
   pointer-events: auto;
   opacity: 1;
+  z-index: 200;
```

Comment:
Looks fine, the find bar has an index of `100` and other relevant elements (such as the project recorder modal) have numbers larger than 200 👍

Reply:
(Besides, the popup would close if you clicked onto anything else but I suppose it would flash in front of it before it closes fully)